http-server-netty/src/test/groovy/io/micronaut/http/server/netty/resources/StaticResourceResolutionSpec.groovy

```diff
@@ -29,13 +28,9 @@ import java.nio.file.Paths
 import java.time.Instant
 import java.time.ZoneId
 import java.time.ZonedDateTime
+import java.time.temporal.ChronoUnit
 
-import static io.micronaut.http.HttpHeaders.CACHE_CONTROL
-import static io.micronaut.http.HttpHeaders.CONTENT_LENGTH
-import static io.micronaut.http.HttpHeaders.CONTENT_TYPE
-import static io.micronaut.http.HttpHeaders.DATE
-import static io.micronaut.http.HttpHeaders.EXPIRES
-import static io.micronaut.http.HttpHeaders.LAST_MODIFIED
+import static io.micronaut.http.HttpHeaders.*
```

Comment:
@wykapedia You should fix your IDE to increase the * to 20 or something larger than 5...

Reply:
only test class, so not huge, but....